### PR TITLE
feat: current resolved as string

### PR DESCRIPTION
**Pull request by sebastiaanviaene**
With this feature you can use the current iteration as a string value in your fixtures by using ``($current)``

````
items:
  user{1..10}:
    email: 'email($current)@gmail.com'

````

Which results in email1@gmail.com .. email10@gmail.com

Additionally you can use math operators to mutate this field

````
items:
  user{1..10}:
    email: 'email($current*10)@gmail.com'

````
Which results in email10@gmail.com .. email100@gmail.com. This gives you total freedom in referencing to other fixtures and using it as a string.

### CODE REVIEW

**Comment by driescroons** [comment]
looks fine for me

File: tslint.json [hunk 1]
```diff
@@ -107,4 +107,4 @@
     ],
     "newline-before-return": true
   }
-}
+}
```
Comment (RobinCK) [inline tslint.json]: please add an empty line

Comment (sebastiaanviaene) [reply tslint.json]: Done

**Comment by RobinCK** [comment]
looks good, please add unit test for this feature and update documentation

File: tslint.json (hunk 1 above)
Comment (RobinCK) [reply tslint.json]: thanks

**Comment by sebastiaanviaene** [comment]
> looks good, please add unit test for this feature and update documentation

Done

**Comment by RobinCK** [comment]
Thank you

**Comment by RobinCK** [comment]
publish typeorm-fixtures-cli@1.2.0

**Comment by RobinCK** [comment]
Thank @sebastiaanviaene 
implemented in typeorm-fixtures-cli@1.2.0

**Comment by sebastiaanviaene** [comment]
> publish typeorm-fixtures-cli@1.2.0

Did you forget to build before publishing? The code in 1.2.0 doesn't seem to have my changes in it. I suggest releasing a "1.2.0-rc.1" with the newly built dist folder.

**Comment by RobinCK** [comment]
@sebastiaanviaene republish typeorm-fixtures-cli@1.2.1

**Pull request merged**